documentation/usersguide/objectdefinitions.md

```diff
@@ -323,6 +323,9 @@ This directive is used to determine whether or not passive checks are enabled fo
 <td valign="top"><strong>check_period</strong>:</td>
 <td>
 This directive is used to specify the short name of the <a href="#timeperiod">time period</a> during which active checks of this host can be made.
+    
+{{ site.note }}If it is not defined, Naemon will check it on a <b><i>as needed basis</i></b>, which can be defined as a 24x7 time frame.  This will also give a warning when running the Naemon check config command.{{ site.end }}
```

Comment:
I am not sure exactly what is meant by "which can be defined as a 24x7 time frame" ?

The behavior is when check_period is 0 is documented here: http://www.naemon.org/documentation/usersguide/hostchecks.html so perhaps we could link to that page for more information.

Reply:
If you look even further back and refer to the Nagios documentation (which the core Naemon features are based upon).  The check_period is 24x7, but that time period is not technically defined.

If you refer to the http://www.naemon.org/documentation/usersguide/hostchecks.html and http://www.naemon.org/documentation/usersguide/servicechecks.html does not even mention the check period, which seems to be odd.

Reply:
I guess the check_period is ignored when on-demand checks are made?

Reply:
As I recall, on-demand checks can be made out the check_period, I think the check_period filters out scheduling checks outside the defined time frame. If there is not a time frame, it just goes ahead and schedules checks.

Reply:
Sorry I completely forget about this PR.

If leaving the `check_period` empty is the same as `24x7`, then perhaps we should just write that in the object definition similar to how we have default values for other objects definitions.